components/sync/driver/BUILD.gn

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import("//brave/build/features.gni")
+import("//brave/components/sync/driver/sources.gni")
```

Comment:
these can be removed too

Reply:
oh, not yet fixed, 3 min

Reply:
fixed at https://github.com/brave/brave-core/compare/4348bf244d9c62618c9632328e1554aec805badd..29d7b51e16014f8c8893f95df8657abe042a1442